AutoSteamApp/Automaton/SteamworkAutomaton.cs

```diff
@@ -0,0 +1,348 @@
+﻿using AutoSteamApp.Configuration;
+using AutoSteamApp.Helpers;
+using AutoSteamApp.ProcessMemory;
+using GregsStack.InputSimulatorStandard;
+using GregsStack.InputSimulatorStandard.Native;
+using Logging;
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Windows.Forms;
+
+namespace AutoSteamApp.Automaton
+{
+
+    public class SteamworkAutomaton
+    {
+
+        #region Fields
+
+        /// <summary>
+        /// Process loaded in the constructor.
+        /// </summary>
+        Process _Process;
```

Comment:
It is a good practice to always declare the access modifiers - that includes private.